frontend/src/components/Comment/CommentReplyForm/CommentReplyForm.tsx

```diff
@@ -66,11 +66,13 @@ function CommentReplyForm<T extends CommentReplyAction>({
 	const { admin } = useAuthContext()
 	const { data: mentionSuggestionsData } =
 		useGetCommentMentionSuggestionsQuery({
-			variables: { project_id },
+			variables: { project_id: project_id! },
```

Comment:
What's up with the trailing `!` here?

Reply:
This occurs in lots of cases and breaks typescript linting, `project_id` from useParams can be undefined (but not in our case, because of the react-router redirects)